### ISSUE TITLE
REPLEX_TMDB_API_KEY

### ISSUE THREAD

**Issue by jl94x4** [sarendsen/replex#19]
I thought I would open a new issue for this instead of using the other thread.

I am using the TMDB for hero style it works, and looks good. However I've noticed that when im on the same page for a while, suddenly all posters on the hero row will revert to my Plex background images. A refresh fixes it, not sure if this is something fixable, or if its something wrong on my end. Here's logs jus incase.

https://pastebin.com/re8mSqyx

**Comment by jl94x4** [comment]
> You’re sure your are on the latest release? Thought I fixed this. Also on which page? Home or a section/library?

Home screen, I pulled latest about an hour ago.

**Comment by jl94x4** [comment]
> Allright, plex probaply does some other refresh calls I’m not aware of. I’ll check it out.
> 
> also you should probaply use the replex-nginx image to prevent high cpu usage

Just changed it, I caught it as I checked if I was on latest :P

**Comment by jl94x4** [comment]
Shows the same behaviour on Roku and Chromecast with Google TV too just thought I'd mention it.

**Comment by jl94x4** [comment]
> How long before it refreshes? Few minutes, 10 minutes?

Less than 10minutes

**Comment by jl94x4** [comment]
Seems to be very hit and miss each refresh, sometimes it works, sometimes not.

**Comment by jl94x4** [comment]
Still seems to he refreshing to the plex backgrounds after so long.

I have updated to the latest release

**Comment by jl94x4** [comment]
Alright I've sent that request.

**Comment by jl94x4** [comment]
Could it be a cache issue?

**Comment by jl94x4** [comment]
> I got it to replicate on ios. Will dive in tomorrow. Looks like it has something todo with the random rows.
> 
> Edit it hasn't got todo with the random rows.

Yeah I did disable random rows and that didn't fix it for me either.

**Comment by jl94x4** [comment]
Did you see it replicate on app.plex.tv? Its doing it for me pretty quickly, just leaving the window open and after so long they all just change, only a hard refresh brings the tmdb posters back. Navigating through libraries via the plex ui they always remain as plex backgrounds.

**Comment by jl94x4** [comment]
Okay, really hope you get this fixed :)

**Comment by jl94x4** [comment]
Something went wrong
There was an error loading your home screen. Please try again in a few minutes

Getting this on ip:port and can't connect at all using app.plex.tv

**Comment by jl94x4** [comment]
Updated and still getting 

`Something` went wrong
There was an error loading your home screen. Please try again in a few minutes.`

**Comment by jl94x4** [comment]
![Screenshot_20230809_100550_Termius](https://github.com/sarendsen/replex/assets/18123855/d258b807-c9e9-4ba5-ac8b-3871d954ec25)

No longer runs, same command I've been running with, too

**Comment by jl94x4** [comment]
![Screenshot_20230809_100801_Termius](https://github.com/sarendsen/replex/assets/18123855/70a622ea-5e9d-47bb-923c-2da6d2900555)

**Comment by jl94x4** [comment]
Seems to be working again. I'll check out the tmdb posters and see if that's also fixed 😁

**Comment by darthShadow** [comment]
Just FYI, you can also use the Plex Metadata API to get the appropriate artwork once you have the Plex GUID & API Token like this:

```
curl -H 'Accept: application/json' 'https://metadata.provider.plex.tv/library/metadata/<guid>?X-Plex-Api-Token=<token>
```

This returns the item metadata from the official Plex servers which also has images classified by type like:

```
        "Image": [
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "background",
            "url": "https://image.tmdb.org/t/p/original/m6eRgkR1KC6Mr6gKx6gKCzSn6vD.jpg"
          },
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "banner",
            "url": "https://artworks.thetvdb.com/banners/graphical/83268-g9.jpg"
          },
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "clearLogoWide",
            "url": "http://assets.fanart.tv/fanart/tv/83268/hdtvlogo/star-wars---the-clone-wars-50f2fc69eb04a.png"
          },
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "coverArt",
            "url": "https://metadata-static.plex.tv/1/gracenote/144b60ca1d28afbebb218bde0d34543e.jpg"
          },
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "coverPoster",
            "url": "https://image.tmdb.org/t/p/original/e1nWfnnCVqxS2LeTO3dwGyAsG2V.jpg"
          },
          {
            "alt": "Star Wars: The Clone Wars",
            "type": "coverSquare",
            "url": "https://metadata-static.plex.tv/a/gracenote/ab097f979f233913e440467f932438f6.jpg"
          }
        ]
```

**Comment by jl94x4** [comment]
This is working brilliantly, now!
